vmm/src/sigwinch_listener.rs

```diff
@@ -205,6 +205,7 @@ fn sigwinch_listener_main(seccomp_filter: BpfProgram, tx: File, tty: File) -> !
 /// Same as [`fork`].
 unsafe fn clone_clear_sighand() -> io::Result<u64> {
     let mut args = clone_args::default();
+    args.exit_signal = SIGCHLD as u64;
```

Comment:
There is a clippy alert here which you should fix.

Reply:
No problem; I'll rewrite this in the way it proposes.